lms/djangoapps/teams/tests/test_api.py

```diff
@@ -122,21 +126,25 @@ def test_is_team_discussion_private_is_public(self):
         assert not teams_api.is_team_discussion_private(None)
         assert not teams_api.is_team_discussion_private(self.team2)
         assert not teams_api.is_team_discussion_private(self.team3)
+        assert not teams_api.is_team_discussion_private(self.team4)
 
     def test_is_instructor_managed_team(self):
         assert teams_api.is_instructor_managed_team(self.team1)
         assert not teams_api.is_instructor_managed_team(self.team2)
         assert teams_api.is_instructor_managed_team(self.team3)
+        assert not teams_api.is_instructor_managed_team(self.team4)
 
     def test_is_instructor_managed_topic(self):
         assert teams_api.is_instructor_managed_topic(COURSE_KEY1, TOPIC1)
         assert not teams_api.is_instructor_managed_topic(COURSE_KEY2, TOPIC2)
         assert teams_api.is_instructor_managed_topic(COURSE_KEY2, TOPIC3)
+        assert not teams_api.is_instructor_managed_topic(COURSE_KEY2, TOPIC4)
```

Comment:
should we add a new test called test_is_open_managed_topic?

Reply:
I don't think so, instructor_managed topics here refers to a team that is controlled by instructors at all levels, however, the new implementation tries to do not conflict with this definition